OpenRA.Mods.Common/Traits/Render/Hovers.cs

```diff
@@ -43,7 +43,9 @@ public IEnumerable<IRenderable> ModifyRender(Actor self, WorldRenderer wr, IEnum
 			if (self.World.Paused || IsTraitDisabled)
 				return r;
 
-			var visualOffset = !aircraft || self.CenterPosition.Z > 0 ? (int)Math.Abs((self.ActorID + Game.LocalTick) / 5 % 4 - 1) - 1 : 0;
+			var visualOffset = !aircraft || self.World.Map.DistanceAboveTerrain(self.CenterPosition).Length > 0
```

Comment:
While I agree with you both here, this pr should have only been about fixing a small oversight, not changing the way hovers works. Adding a `MinHoveringAltitude` will require yaml modifications.